lib/src/nyxx/internal/Http.dart

```diff
@@ -285,8 +285,11 @@ class Http {
   Logger _logger = Logger.detached("Http");
 
   Http._new() {
-    this._headers= {'User-Agent':
-        'DiscordBot (https://github.com/l7ssha/nyxx, ${_Constants.version})'};
+    if(!browser)
```

Comment:
"Clients using the HTTP API must provide a valid User Agent". [See](https://discordapp.com/developers/docs/reference#user-agent)

Reply:
Yikes! That is impossible for a web script. I will ask about this in discord api chat.